app/components/member/Teams/chooser.js

```diff
@@ -98,25 +100,23 @@ class Chooser extends React.Component {
   }
 
   handleDialogOpen = () => {
-    // Fetch the teams list on first open by setting (if: $showingDialog) to true
-    // and force re-fetch on subsequent opens in case the teams list has changed
-    this.props.relay.forceFetch({
-      showingDialog: true,
-      userSelector: `!${this.props.organizationMember.user.uuid}`
-    }, (state) => {
-      // When we're finished re-fetching, show the dialog
+    // First switch the component into a "loading" mode and refresh the data in the chooser
+    this.setState({ loading: true });
+    this.props.relay.forceFetch({ isMounted: true, userSelector: `!${this.props.organizationMember.user.uuid}` }, (state) => {
       if (state.done) {
-        this.setState({
-          showingDialog: true
-        }, () => {
-          this._autoCompletor.focus();
-        });
+        this.setState({ loading: false });
       }
     });
+
+    // Now start showing the dialog, and when it's open, autofocus the first
+    // result.
```

Comment:
Lol, this is a bad comment - I meant to say focus the search input. Will fix my comments!